ipwb/util.py

```diff
@@ -146,6 +146,14 @@ def iso8601ToDigits14(iso8601DateString):
     return d.strftime('%Y%m%d%H%M%S')
 
 
+def isRFC1123Compliant(dtstr):
```

Comment:
@machawk1 I think this method is wasteful, duplicates the code, and causes the same logic to be executed twice. This same effect could have been achieved by catching the `ValueError` exception in the `replay.py` file when calling `ipwbUtils.rfc1123ToDigits14(adt)` function.

Reply:
@ibnesayeed I anticipate this function to be useful elsewhere, hence the inclusion in util and not hard-coded to the use case in replay.

Reply:
To deal with parsing errors like this, exceptions are generally a better way, no matter how often you use them.